packages/tools/src/types/AnnotationTypes.ts

```diff
@@ -1,6 +1,42 @@
 import type { Types } from '@cornerstonejs/core';
 
-type Annotation = {
+export type AnnotationMetadata = Types.ViewReference & {
+  /**
+   * The registered name of the tool
+   */
+  toolName: string;
+  /**
+   * The position of the camera in world space.
+   */
+  cameraPosition?: Types.Point3;
+  /**
+   * The viewUp for the view position
+   */
+  viewUp?: Types.Point3;
+
+  /** The color of the segmentation - really this is annotation data */
+  segmentColor?;
+};
+
+export type AnnotationData = {
+  /** Annotation handles that are grabbable for manipulation */
```

Comment:
I'm not sure if the apidocs understand /** same line */ if it is same line use //